endtoend/evaluators/operations.go

```diff
@@ -23,17 +24,23 @@ var exitedIndice uint64
 // valExited is used to know if exitedIndice is set, since default value is 0.
 var valExited bool
 
+// churnLimit is normally 4 unless the validator set is extremely large.
+var churnLimit = uint64(4)
+var depositValCount = params.E2ETestConfig().MinGenesisActiveValidatorCount / 4 // Dividing by 4 for 4 beacon nodes in long running e2e.
```

Comment:
can the 4 be the num beacon nodes variable?

Reply:
I tried it but I think the code that declares the amount of beacon nodes isn't ready yet since this is something that needs to be exported.